--- origsrc/expat-2.0.1/configure.in	2007-05-09 08:26:48.000000000 -0500
+++ src/expat-2.0.1/configure.in	2008-05-04 08:37:45.765625000 -0500
@@ -30,6 +30,7 @@
 undefine([expat_version])
 
 AC_CONFIG_SRCDIR(Makefile.in)
+AC_CONFIG_MACRO_DIR(conftools)
 AC_CONFIG_AUX_DIR(conftools)
 
 
@@ -62,6 +63,7 @@
 
 dnl Checks for programs.
 AC_PROG_CC
+AC_PROG_CXX
 AC_PROG_INSTALL
 
 if test "$GCC" = yes ; then
